testserv.c

```diff
@@ -123,6 +153,9 @@ mustforksrv()
     /* now in child */
 
     prot_init();
+#ifdef WITHCOV
```

Comment:
I think it's reasonable to install the signal handler unconditionally, especially if it uses `SIGTERM`.

Reply:
I was meaning to use SIGTERM too because I think that this case should be handled as generally as possible.